src/Components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,277 @@
+  events,
+  backgroundEvents,
+  availability,
+  defaultEventLength = defaultEveLength,
+  athletes,
+}: Props) => {
+  const [date, setDate] = useState<Date>(moment(new Date()).toDate());
+  const [view, setView] = useState<(typeof Views)[Keys]>(Views.MONTH);
+  const [toggleValue, setToggleValue] = useState(defaultView);
+  const [myEvents, setEvents] = useState<CalendarEvent[]>(events);
+  const [selected, setSelected] = useState();
+  const [eventPopup, setEventPopup] = useState<PopupEvent>();
+  const [show, setShow] = useState(false);
+
+  const currentMonth = date.toLocaleString("default", { month: "long" });
+
+  const toolbarTitle = new Map();
+  toolbarTitle.set("month", `${currentMonth} ${date.getFullYear()}`);
+  toolbarTitle.set("week", `${currentMonth} ${date.getFullYear()}`);
+  toolbarTitle.set(
+    "day",
+    `${date.getDate()}-${date.getMonth() + 1}-${date.getFullYear()}`
+  );
+
+  class CustomToolbar extends Toolbar {
+    handleNavigation = (value: string) => {
+      switch (value) {
+        case "Previous":
+          this.navigate("PREV");
+          break;
+        case "Today":
+          this.navigate("TODAY");
+          break;
+        case "Next":
+          this.navigate("NEXT");
+          break;
+      }
+    };
+
+    render() {
+      return (
+        <div className="rbc-toolbar">
+          <div className={styles.toolbarNav}>
+            <Toggle
+              name="NavButtons"
+              values={["Previous", "Today", "Next"]}
+              defaultValue=""
+              setChecked={this.handleNavigation}
+            />
+          </div>
+          <p className="rbc-toolbar-label">{this.props.label}</p>
+          <Toggle
+            name="viewSelect"
+            values={["Month", "Week", "Day"]}
+            setChecked={changeView}
+            defaultValue={toggleValue}
+          />
+        </div>
+      );
+    }
+
+    navigate = (action) => {
+      this.props.onNavigate(action);
+    };
+  }
+
+  const onView = useCallback(
+    (newView) => {
+      setView(newView);
+      setToggleValue("Day");
+    },
+    [setView]
+  );
+
+  //Week Label
+  const dayRangeHeaderFormat = (
+    { start, end },
+    culture,
+    localizer: momentLocalizer
+  ) => {
+    const sameMonth = localizer.eq(start, end, "month");
+    const sameYear = localizer.eq(start, end, "year");
+    const formatter = `DD ${sameMonth ? "" : "MMM"} ${sameYear ? "" : "YYYY"}`;
+
+    return (
+      localizer.format(start, formatter, culture) +
+      " – " +
+      localizer.format(
+        end,
+        localizer.eq(start, end, "month") ? "DD MMMM YYYY" : "DD MMM YYYY",
+        culture
+      )
+    );
+  };
+
+  //Day Label
+  const dayHeaderFormat = (date: Date, culture, localizer: momentLocalizer) =>
+    localizer.format(date, "dddd DD MMMM, YYYY", culture);
+
+  const changeView = (value: string) => {
+    if (value === "Month") {
+      setView(Views.MONTH);
+      setToggleValue("Month");
```

Comment:
Better to just set toggle value to `value` instead of hard coding "Month" here. If we ever change those values in the future, we'd want to just do it once